languages/clojure/exercises/concept/basic-functions/.doc/after.md

```diff
@@ -0,0 +1,13 @@
+## Key Learnings
+
+- Functions are defined with the `defn` keyword.
+- Functions can receive one or more arguments.
+- Functions have docstrings which allow you to describe the function in words. 
+- Once a function is defined, it can be invoked or imported elsewhere in the code.
+- The result of the last expression of a Clojure function is the value i returns.
```

Comment:
```suggestion
- The result of the last expression of a Clojure function is the value it returns.
```

Reply:
In Clojure, "keyword" can have a different meaning. Maybe change or omit that piece?